rust/kernel/file_operations.rs

```diff
@@ -256,6 +295,7 @@ unsafe extern "C" fn unlocked_ioctl_callback<T: FileOperations>(
         let f = unsafe { T::Wrapper::borrow((*file).private_data) };
         let mut cmd = IoctlCommand::new(cmd as _, arg as _);
         let ret = T::ioctl(&f, unsafe { &FileRef::from_ptr(file) }, &mut cmd)?;
+        unsafe { pr_info!("the file and oob_data after unlocked_ioctl is {:p}, {:p}", file, (*file).oob_data) };
```

Comment:
Is the `pr_info` here used when you debug? You can consider replacing it with `pr_debug` or deleting it. And the `use::crate::pr_info` at the beginning of the file.

Reply:
Thank you so much! I will delete this line.